packages/angular/cli/commands/add.md

```diff
@@ -1,2 +1,3 @@
-Add support for a library in your project, for example adding `@angular/pwa` which would configure
-your project for PWA support.
+Adds the npm package for a published library to your workspace, and configures your default
```

Comment:
Tech question: Configures the app set in "defaultProject" or configures the current app project (when executed from within an app project folder)?

Reply:
I don't think there is any "current app project" here - it doesn't take a "project" arg, just the package name. I added a note about "defaultProject".